test/settingsLoader_test.js

```diff
@@ -18,37 +19,37 @@ describe('Settings', function(){
     });
 
     it('will default to default port', function(done) {
-        settingsLoader.load('/test/settings/noPort.json',function(err,res){
+        settingsLoader.load(path.normalize('/test/settings/noPort.json'),function(err,res){
```

Comment:
normalize does NOT do what you think it does (it doesn't automagically turn path separators into platform specific path separators):

https://nodejs.org/api/path.html#path_path_normalize_path